pkg/transport/timeout_dialer.go

```diff
@@ -20,17 +20,17 @@ import (
 )
 
 type rwTimeoutDialer struct {
-	wtimeoutd  time.Duration
-	rdtimeoutd time.Duration
+	writeTimeout time.Duration
+	readTimeout  time.Duration
 	net.Dialer
 }
```

Comment:
The original commit upstream didn't change the names of the fields in this structure.

Reply:
yep good catch no sense being greedy.